### PR TITLE
mocha: first stab at automated headless testing using phantomjs + mocha

### PR DESCRIPTION
**Pull request by talos**

### CODE REVIEW

File: static/test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+            describe("the search button", function () {
+
+                it("has the text 'Lookup'", function() {
+                    expect($('#searchbox * .search-button').text()).to.be('Lookup');
+                });
+            });
+        });
+    });
+})();
```
Comment (antoviaque) [inline static/test/index.test.js:54]: jslint gives: `Move the invocation into the parens that contain the function.`

**Comment by antoviaque** [comment]
Looks pretty neat! Should be cleaner than the excessive stubbing needed for QUnit.

Btw I've updated the installation instructions for Debian/Ubuntu, as the default phantomjs 1.6.0 wasn't running the tests correctly.

**Comment by antoviaque** [comment]
Ah and also - the homepage shows `#mocha div missing, add it to your document`

File: static/test/index.test.js (hunk 1 above)
Comment (talos) [inline static/test/index.test.js:54]: Weird, I remember getting that warning from JS Lint before, but for some reason it's not popping up for me now.  Will fix, though!

**Comment by talos** [comment]
Fixed the points you made and opened a new pull request:

https://github.com/antoviaque/languagelearning/pull/3

I think it might be better if we hold off on merging pull requests 'til it's clean from the person who opened it (yours truly!)  The only thing you noticed as really wrong before merging was the JS Lint hiccup, but it's still better (in my mind) to avoid even having that kind of mistake on master.

**Pull request merged**